### ISSUE TITLE
in PUT v2/entities/<id>/attrs/<name>/value if name has # character is returned 405 without payload

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#2083]
in `PUT v2/entities/<id>/attrs/<name>/value` if `id`  has # character is returned 405 without payload

#### update request
```
PUT http://localhost:1026/v2/entities/house_#/attrs/temperature/value
Content-Type: text/plain
Fiware-Service: test_update_entity_id
Fiware-ServicePath: /test
payload: "80"
```
#### update response
```
http code: 405 date: Mon, 25 Apr 2016 16:54:53 GMT
fiware-correlator: 6eae1a84-0b06-11e6-911b-005056a20feb
connection: Keep-Alive
content-length: 0
allow: GET, DELETE
```

#### expected response
```
http code: 405
...
{"error": "methodNotAllowed", "description": "not defined yet"}
```

**Comment by fgalan** [comment]
`#` is a fobidden char in entity ids, so I think that "expected response" shouldn't be a 405 Method Not Allowed, but the one we use in the others cases for fobbiden characteres in ids.

**Comment by kzangeli** [comment]
We don't get that far as to see 'forbidden char' as the request is stopped before by the '405 Method not allowed'.
The problem about no payload for 405 is fixed by PR #2318  (see Issue #2075)

**Comment by fgalan** [comment]
Fixed by PR #2318. Please @iariasleon have a look and close it if you find it solved.

**Comment by iariasleon** [comment]
LGTM
```
http code: 405 - Method Not Allowed
headers:
   content-length: 63
   fiware-correlator: d7235ca8-4e6a-11e6-a1f1-005056a20feb
   connection: Keep-Alive
   allow: GET, DELETE
   date: Wed, 20 Jul 2016 11:12:26 GMT
   content-type: application/json
payload: {"error":"MethodNotAllowed","description":"method not allowed"}
```
